### PR TITLE
[installer] Do not pull blobserve implementation into installer

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

By referencing the `blobserve/pkg/blobserve`, we were effectively pulling all of blobserve (and it's dependencies) directly as a dependency into the installer.

This change moves the necessary config into the `blobserve/config` such that we only depend on that, and not on the blobserve implementation packages (see removed deps in installer).

This in turn does the following:
* Installer doesn't know about blobserve implementation details (from packaging perspective)
* Installer builds faster

## Related Issue(s)

Fixes #

## How to test

CI passes

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/blobserve/pkg/blobserve/blobserve.go [hunk 1]
```diff
@@ -44,40 +44,6 @@ type BlobserveInlineVars struct {
 	SupervisorImage string `json:"supervisor"`
 }
 
-type BlobSpace struct {
```
Comment (easyCZ) [inline components/blobserve/pkg/blobserve/blobserve.go:47]: This is moved to the `config` package

**Comment by akosyakov** [comment]
I wonder should not we separate implementation from api via dedicated component like `components/blobserve-api`. I think we follow this pattern in all other components, but overlooked here because of historical reasons. It would separate cleanly go deps?

**Review by akosyakov — APPROVED** [review]
I think it makes sense, looks like a tech debt.

Not sure whether we should rather separate it on level of leeway components instead as we do for other components. 

/hold

if you decide to refactor it once more

**Comment by easyCZ** [comment]
Given the necessary dependencies are already not included with this change, I don't think we necessarily need to introduce the api package for this. In some cases it could make sense, but here it'd just hold the config which seems unnecessary

**Pull request merged**